cogboard-webapp/src/components/widgets/types/WorldClockWidget/index.js

```diff
@@ -1,24 +1,41 @@
 import React, {useState} from 'react';
-import Typography from "@material-ui/core/Typography/index";
+import {string} from "prop-types";
+import {Typography} from "@material-ui/core";
```

Comment:
Move to components imports. I remind that you have 3 types of imports in the following order: libs, internal helpers/actions/hooks, components: 3rd party/own.